src/common.py

```diff
@@ -410,6 +410,9 @@ def __init__(self):
         self.__space = os.getenv('CONFLUENCE.SPACE')
         self.__confluence = Confluence(url=confluence_url, token=confluence_token)
 
+    def get_confluence(self):
+        return self.__confluence
```

Comment:
Remove this. We do not want a class to directly access atlassian.Confluence but only through this proxy class. If you need functionality, it should be added to this class.

Reply:
This method is not used anymore and is now deleted